enos/modules/ec2_info/main.tf

```diff
@@ -25,6 +34,19 @@ locals {
         "20.04" = data.aws_ami.ubuntu_2004["x86_64"].id
         "22.04" = data.aws_ami.ubuntu_2204["x86_64"].id
       }
+      "amazon_linux" = {
+        "amzn2" = data.aws_ami.amazon_linux_2["x86_64"].id
+      }
+      "leap" = {
+        "15.4" = data.aws_ami.leap_154["x86_64"].id
+        "15.5" = data.aws_ami.leap_155["x86_64"].id
+      }
+      "sles" = {
+        "v12_sp4_standard" = data.aws_ami.sles_12_sp4_standard.id
```

Comment:
Did we get a chance to launch an instance with any of the SLES AMI's? Since they're published to the AWS Marketplace I was hoping to determine whether or not we'll have to manually "subscribe" to the AMI's before they're used. If that's the case we'd want to use very specific images here rather than latest so as to not break the pipeline if/when they cut a new AMI.

Reply:
Yes, I've successfully launched all of them now. The openSUSE Leap ones require a subscription (one for amd64 images, one for arm64 images) and I've included that info in a comment.